src/pages/EmployeeCreation/EmployeeCreationPage.jsx

```diff
@@ -0,0 +1,121 @@
+import { Avatar, Box, Button, Fab, FormGroup, TextField, Typography } from "@mui/material";
+import UploadBox from "./components/UploadBox";
+import PageLayout from "../../components/PageLayout";
+import { useState } from "react";
+
+/**
+ * The `EmployeeCreationPage` function is a React component that renders a form for users to create
+ * a new employee, including fields for the employee's name, email, phone number, address, and
+ * additional information, as well as a button to upload a resume and a button to create the employee.
+ * @returns {JSX.Element} The Employee creation page component.
+ */
+function EmployeeCreationPage() {
+
+  const [employeeName, setEmployeeName] = useState("");
+  const [employeeEmail, setEmployeeEmail] = useState("");
+  const [employeePhoneNumber, setEmployeePhoneNumber] = useState("");
+  const [employeeAddress, setEmployeeAddress] = useState("");
+  const [employeeExtraInfo, setEmployeeExtraInfo] = useState("");
+
+  const handleEmployeeNameChange = (event) => {
+    setEmployeeName(event.target.value);
+  };
+  const handleEmployeeEmailChange = (event) => {
+    setEmployeeEmail(event.target.value);
+  };
+  const handleEmployeePhoneNumberChange = (event) => {
+    setEmployeePhoneNumber(event.target.value);
+  }
+  const handleEmployeeAddressChange = (event) => {
+    setEmployeeAddress(event.target.value);
+  }
+  const handleEmployeeExtraInfoChange = (event) => {
+    setEmployeeExtraInfo(event.target.value);
+  }
+
+  function handleSubmit(event) {
+    console.log("Employee has been created");
+    console.log("Name of employee: " + employeeName);
+    console.log("Email of employee: " + employeeEmail);
+    console.log("Phone number of employee: " + employeePhoneNumber);
+    console.log("Address of employee: " + employeeAddress);
+  }
+  function changeAvatar(event) {
+    console.log("Avatar has been changed");
+  }
+
+
+
+
+  return (
+    <PageLayout viewportPage title="Employee Creation">
+      <FormGroup className="employeeCreationForm">
+        <Box
+          sx={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            minHeight: "100vh",
+            minWidth: "100vw",
+            flexDirection: "column"
+          }}
+        >
+          <Box maxWidth="50%">
+            <Box display="flex" flexDirection="row" alignItems="center" justifyContent="center">
+              <Avatar sx={{ width: "10vw", height: "20vh", variant: "circular" }} />
```

Comment:
<img width="782" alt="image" src="https://github.com/SE2-WS23/Skillverse-Web/assets/65715000/45a4b189-3538-40fb-8f02-fd0df8496cba">

Reply:
It should stay rounded

Reply:
Yeah, that would be important